src/functions/asyncCredential/sessionService/sessionService.ts

```diff
@@ -64,6 +66,49 @@ export class SessionService implements IRecoverAuthSession {
     return successResponse(result.Items[0].sessionId.S);
   }
 
+  async createSession(
+    sessionConfig: ISessionConfig,
+  ): Promise<ErrorOrSuccess<null>> {
+    const config = {
+      TableName: this.tableName,
+      Item: {
+        authSessionId: { S: sessionConfig.authSessionId },
```

Comment:
do we want to call this authSessionId? or just sessionId?

Reply:
In the (other) BE, we have multiple different types of sessions and it made it confusing when seeing sessionId, biometricSessionId etc etc, so we though authSessionId from the outset would be useful just incase other types of sessions were ever introduced in this flow

Reply:
Updated